podwatcher/podwatcher.go

```diff
@@ -155,7 +155,7 @@ func (cl *ContainerList) UpdateContainer(c *Container) error {
 }
 
 func ExtractContainersFromPod(pod *corev1.Pod) map[string]*Container {
-	sourceType, ok := pod.GetLabels()["source_type"]
```

Comment:
I'm just a bit concerned if we should actually do this in a backward-compatible manner, so nobody has to figure out the versioning, or either document that this change is not compatible with earlier versions of Eirini

Reply:
I'm pretty sure we are the only consumers of this for now. Also if there are any other consumers, they probably use it through the eirini bosh release, in which we are going to bump the eirini submodule too. Making it backward compatible will require unreasonably complicated code (matching labels etc). I think we are good like this.